weblab_cg/_printer.py

```diff
@@ -163,6 +161,13 @@ def _print_And(self, expr):
         my_prec = precedence(expr)
         return ' and '.join([self._bracket(x, my_prec) for x in expr.args])
 
+    def _print_bool(self, expr):
+        """ Handles False """
```

Comment:
It doesn't just handle `False`

Reply:
No maybe more useful to add that it's for python bools (done)